test/unit/app/renderer/downloadItemTest.js

```diff
@@ -53,13 +50,19 @@ describe('downloadItem component', function () {
       before(function () {
         this.downloadId = uuid.v4()
         this.download = newDownload(state)
-        this.result = mount(<DownloadItem downloadId={this.downloadId} download={this.download} downloadDeleteConfirmationVisible={this.download.downloadDeleteConfirmationVisible} />)
+        this.result = mount(
+          <DownloadItem
+            downloadId={this.downloadId}
+            download={this.download}
+            deleteConfirmationVisible={this.download.get('deleteConfirmationVisible')} />
```

Comment:
Thanks! I never worked with ImmutableJS before, so most of my code in there was just a blind guess.